### PR TITLE
Billions Must Flip

### PR DESCRIPTION
**Pull request by sleepyyapril**
# Description

Billions must flip.

https://github.com/Goob-Station/Goob-Station/pull/828
https://github.com/Goob-Station/Goob-Station/pull/832

# Changelog

:cl:
- add: Spin, flip, and jump emotes have been added.

### CODE REVIEW

**Review by Remuchi — APPROVED** [review]

File: Resources/Prototypes/Actions/emotes.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+- type: emote
+  id: Flip
+  name: chat-emote-name-flip
+  chatMessages: ["chat-emote-msg-flip"]
+  chatTriggers:
+  - flip
```
Comment (Mnemotechnician) [inline Resources/Prototypes/Actions/emotes.yml:6]: These need to be updated, we have message suffix trimming on EE, so you don't have to duplicate every chat trigger (iirc that is)

File: Resources/Prototypes/Entities/Mobs/base.yml [hunk 2]
```diff
@@ -46,6 +46,14 @@
...
+  - type: AnimatedEmotes
+
+- type: entity
+  save: false
+  id: MobPolymorphable
+  abstract: true
+  components:
+  - type: Polymorphable
   - type: OwnInteractionVerbs
     allowedVerbs: [] # TODO: define something here, or don't.
```
Comment (Mnemotechnician) [inline Resources/Prototypes/Entities/Mobs/base.yml:58]: This should belong to BaseMob as it always did.

**Review by Mnemotechnician — COMMENTED** [review]
Crazy

File: Resources/Prototypes/Entities/Mobs/base.yml (hunk 2 above)
Comment (sleepyyapril) [reply Resources/Prototypes/Entities/Mobs/base.yml:58]: That is the point, yes.

**Comment by sleepyyapril** [comment]
Just to be clear: mobs being able to flip is intentional.

**Review by VMSolidus — APPROVED** [review]

File: Resources/Prototypes/Entities/Mobs/base.yml (hunk 2 above)
Comment (Mnemotechnician) [reply Resources/Prototypes/Entities/Mobs/base.yml:58]: That is not the point, you nuked interaction verbs on all mobs by doing that

Comment (sleepyyapril) [reply Resources/Prototypes/Entities/Mobs/base.yml:58]: That's crazy

**Pull request merged**